gsuite_reports_rules/gsuite_doc_ownership_transfer.py

```diff
@@ -1,4 +1,4 @@
-from panther_base_helpers import gsuite_parameter_lookup as param_lookup
+from panther_base_helpers import gsuite_parameter_lookup as param_lookup  # pylint: disable=import-error
```

Comment:
I added these just to clean up my editor :)

Reply:
I'd go so far as to say we could just update our pylint rules in the makefile to always ignore `import-error`, it's never found anything useful and these comments are weird to see in Panther.